src/Tools/TransformXMLToJSONUsingXSLT/README.md

```diff
@@ -0,0 +1,76 @@
+# Transformation Tool from XML in format NUnit2 to Text
+
+Transformation Tool is an application to:.
+
+  - Transform files XML to Text
+  - Create text files
+  - Configure the entry files and set the out put files.
+
+### Technologies
+
+ - .Net FrameWork 4.7 
+ - XSLT Files to Transform.
+ - C# as language
+
+### Configuration
+
+Transformation Tool needs the next entries to work properly, this entries have to be into app.config file and they are in two sections which are mentioned below.
+
+**appSettings**
+This section is described in the table Files needed above.
+```sh
+<appSettings>
+    <add key="xmlFilePathResult" value="TestResult-Color_Tests.xml" />
+    <add key="xsltFile" value="TransformXMLToText.xslt" />
+    <add key="textFile" value="CustomTextResult.txt" />
+    <add key="textFileWithFilters" value="TextResultWithFilters.txt" />
+    <add key="applyFilters" value="1" /><!--1 means yes, 0 means not-->
+</appSettings>
+```
+**About TestResult-Color_Tests.xml**
+This file is in format NUnit2 and it is the result of a set of running test, there is an example above:
+Enter to the path where the console of NUnit where installed by default C:\Program Files (x86)\NUnit.org\nunit-console and the type the next command.
+```
+nunit3-console.exe "C:\\DSCoreNodesTests.dll" --where "test=="DSCoreNodesTests.ColorTests" and cat != Failure and cat != BackwardIncompatible" --labels=Before --explore="C:\\TestResult-Color_Tests.xml";format=nunit2
```

Comment:
I think you should *not* assume that nunit3 runner is installed - can you look at pulling it as part of the build:
something like this perhaps?
https://www.nuget.org/packages/NUnit.Console/

@alfredo-pozo @smangarole  @QilongTang please consider if this should be done in this PR or as part of the CI build job instead.

Reply:
This was just a sample to show how can we get the test result file.